docs/product/accounts/migration/index.mdx

```diff
@@ -16,65 +16,72 @@ For additional reading on considering SaaS, take a look at:
 - [XS:Code > Open Source is Free - But Expensive](https://dev.to/helad11/open-source-is-free-but-expensive-3h8a)
 - [Frequently asked migration questions](https://sentry.io/from/self-hosted/)
 
-## Migrating to Cloud
+## Relocating to Cloud
 
-If your organization started with self-hosted Sentry, you've invested time and effort in configuring and customizing your Sentry org. Once you decide to move to a SaaS account, it's important to know that you can migrate a lot of that investment into your new account instead of starting from scratch.
+If your organization started with self-hosted Sentry, you've invested time and effort in configuring and customizing your Sentry org. Once you decide to move to a SaaS account, it's important to know that you can relocate a lot of that investment into your new account instead of starting from scratch.
 
-To make this migration as seamless as possible, Sentry has a migration workflow that allows you to migrate most of your account's settings and configurations.
+To make this relocation as seamless as possible, Sentry has a workflow that allows you to retain most of your account's settings and configurations as you move to SaaS. All you'll need is owner access to your self-hosted install, and a way to access it via the terminal.
 
-One thing to note is we won't be able to transfer over historical event data, but new incoming events will populate the projects in your new organization. You will also need to re-invite all your members and add them to the correct teams.
+### What you'll keep
 
-## How to Migrate
+All of the configurations, team and project structure, and most metadata will be relocated. Specifically, for every organization you [choose to relocate](#2-choose-which-organizations-to-relocate) this includes:
 
-### 1. Sign up
+- All of the organization settings
+- The user accounts and options for all of the organization's members
+- Projects and their settings
+- Teams and their settings
+- Alert rules
+- Crons
+- Dashboards
+- Enabled data scrubbers
+- Repositories
+- Recent and saved searches
 
-Before you begin the migration process, [sign up](https://sentry.io/signup/) for a new organization in Sentry.
+### What gets left behind
 
-![Sentry welcome page](./img/sign-up-welcome.png)
+Any data related to specific issues, events, stack traces, or files like replays and source maps won't be coming along. Additionally, for security reasons, you'll need to reconnect and re-authenticate all of your integrations, and your users will need to re-connect their 2FA and mint new API tokens after relocating.
```

Comment:
```suggestion
Any data related to specific issues, events, stack traces, or files like replays and source maps won't be transferred. Additionally, for security reasons, you'll need to reconnect and re-authenticate all of your integrations, and your users will need to re-connect their 2FA and create new API tokens after relocating.
```

We try to avoid language that may not be as easy to understand for non native English speakers :)